src/components/tasks/CreateTasks.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTasks } from "src/store/actions/taskActions";
+import "src/components/tasks/CreateTasks.component.scss";
+import { createDate } from "src/utils/helpers/createDate";
+import PropTypes from "prop-types";
+
+const CreateTasks = ({ show, onShow }) => {
+  const dispatch = useDispatch();
+  const [task, setTask] = useState("");
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  const handleChange = (e) => {
+    setTask(e.target.value);
+  };
```

Comment:
Please use full descriptive name for these. and also avoid anonymous function in all files.
```suggestion
  function handleChange(event){
    setTask(event.target.value);
  };
```